internal/app/secretless/providers/conjur/provider.go

```diff
@@ -105,8 +102,15 @@ func ProviderFactory(options plugin_v1.ProviderOptions) (plugin_v1.Provider, err
 		}
 		provider.Authenticator = authenticator
 
-		// Kick off the goroutine that will maintain the Conjur access token
-		go provider.refreshAccessToken()
+		refreshErr := provider.refreshAccessToken(true)
+		if refreshErr != nil {
+			return nil, refreshErr
+		}
+
+		go func() {
+			time.Sleep(provider.Authenticator.Config.TokenRefreshTimeout)
+			provider.refreshAccessToken(false)
+		}()
```

Comment:
Is the `()` at the end of this line hanging?

Reply:
nope - it's needed for the golang func to start running

Reply:
nope - it's needed for the golang func to start running